Sources/MapboxDirections/RouteLeg.swift

```diff
@@ -281,13 +281,19 @@ open class RouteLeg: Codable {
     open var expectedTravelTime: TimeInterval
 
     /**
-     :nodoc:
      The administrative regions through which the leg passes.
-          
+     
+     Items are ordered by appearance, most recent one is at the beginning.
      This property is set to `nil` if no administrative region data is available.
      */
```

Comment:
Since we’re making this type public, its documentation should refer to the `Intersection.regionCode` property as an alternative.